mkdocs.yml

```
@@ -394,6 +394,9 @@ extra:
   product_name_connect: 'Ibexa Connect'
   product_name_cloud: 'Ibexa Cloud'
   product_name_oss: 'Ibexa OSS'
+
+  eol_versions: ["2.5"]
```

I'm disagree with mixing up single and double quotes.

```suggestion
  eol_versions: ['2.5']
```